go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/kaleido-io/kaleido-sdk-go
+module github.com/alexinthesky/kaleido-sdk-go
```

Comment:
Could you revert this change (which I understand was for your local development)